src/Routing/RouteBuilder.php

```diff
@@ -612,7 +612,12 @@ public function prefix($name, $params = [], callable $callback = null)
             $params = [];
         }
         $name = Inflector::underscore($name);
-        $path = '/' . $name;
+        if (isset($params['path'])) {
+            $path = ($params['path'][0] === '/' ? '' : '/') . $params['path'];
```

Comment:
If the developer is passing a path, they can just do the right thing. I don't think we should be forcing a `/` on people.